lib/ModelMixins/AutoRefreshingMixin.ts

```diff
@@ -3,44 +3,67 @@ import {
   IReactionDisposer,
   onBecomeObserved,
   onBecomeUnobserved,
-  reaction
+  reaction,
+  autorun
 } from "mobx";
 import { now } from "mobx-utils";
 import Constructor from "../Core/Constructor";
 import Model from "../Models/Model";
-import MappableTraits from "../Traits/MappableTraits";
+import AutoRefreshingTraits from "../Traits/AutoRefreshingTraits";
+import AsyncMappableMixin from "./AsyncMappableMixin";
 
-type AutoRefreshing = Model<MappableTraits>;
+type AutoRefreshing = Model<AutoRefreshingTraits>;
 
 export default function AutoRefreshingMixin<
   T extends Constructor<AutoRefreshing>
 >(Base: T) {
-  abstract class AutoRefreshingMixin extends Base {
-    _autoRefreshDisposer: IReactionDisposer | undefined;
+  abstract class AutoRefreshingMixin extends AsyncMappableMixin(Base) {
+    private autoRefreshDisposer: IReactionDisposer | undefined;
+    private autorunRefreshEnableDisposer: IReactionDisposer | undefined;
 
-    /* Return the interval in seconds to poll for updates. */
+    /** Return the interval in seconds to poll for updates. */
```

Comment:
You have very specific code comment formatting :)

Reply:
haha that is just for JSDoc/Vscode - otherwise i don't get the description when I hover over it